docs/content/quickstart/parameters.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "QuickStart: Parameters"
+descriptions: Learn how to use a bundle with parameters
+layout: single
+---
+
+Now that you know how to install a bundle, let's look at how to specify parameters to customize how that bundle is installed.
```

Comment:
Now that you know how to install a bundle, let's look at how to customize the bundle installation with parameters.